packages/core/src/amazonqFeatureDev/util/files.ts

```diff
@@ -115,9 +119,13 @@ export async function collectFiles(
                         zipFilePath: prefixWithFolderPrefix(relativePath.workspaceFolder, relativePath.relativePath),
                     })
                 }
+                totalSize += fileStat.size
+                if (totalSize > maxRepoSize) {
+                    throw new ContentLengthError()
```

Comment:
What does the customer see when this happens?

Reply:
"The project you have selected for source code is too large to use as context. Please select a different folder to use for this conversation"

Reply:
For reference
![image](https://github.com/aws/aws-toolkit-vscode/assets/1529554/579b2091-6bfe-4be0-a32b-fcef9c030928)